### PR TITLE
Benchmark for on_container_author_noted only

### PR DESCRIPTION
**Pull request by nanocryk**
This new bench will allow us to get a better idea of how much 1 author noting progress costs, and thus better estimate how much we can support in one block.

Do you see any issue removing the condition over `runtime-benchmarks` features in the runtimes? `set_latest_author_data` will change weight but as it is Mandatory the weight hint doesn't matter IIRC.

### CODE REVIEW

File: pallets/author-noting/src/benchmarks.rs [hunk 1]
```diff
@@ -137,6 +137,14 @@ benchmarks! {
         assert_ok!(Pallet::<T>::set_author(RawOrigin::Root.into(), para_id, block_number, author, u64::from(block_number).into()));
     }: _(RawOrigin::Root, para_id)
 
+    on_container_author_noted {
```
[the remark this answers is not in the document]
Comment (nanocryk) [reply pallets/author-noting/src/benchmarks.rs:140]: Yes I'll run it on my machine for now.

[the remark this answers is not in the document]
Comment (tmpolaczyk) [reply pallets/author-noting/src/benchmarks.rs:147]: Will you push the weights to the PR @nanocryk ?

File: tmp/flashbox_weights/pallet_author_noting.rs [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+	/// Storage: `System::Account` (r:101 w:101)
+	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
+	/// Storage: `Invulnerables::Invulnerables` (r:1 w:0)
+	/// Proof: `Invulnerables::Invulnerables` (`max_values`: Some(1), `max_size`: Some(6402), added: 6897, mode: `MaxEncodedLen`)
+	/// The range of component `x` is `[1, 100]`.
+	fn set_latest_author_data(x: u32, ) -> Weight {
+		// Proof Size summary in bytes:
+		//  Measured:  `854 + x * (208 ±0)`
+		//  Estimated: `7887 + x * (2603 ±0)`
+		// Minimum execution time: 83_801_000 picoseconds.
```
Comment (girazoki) [inline tmp/flashbox_weights/pallet_author_noting.rs:90]: you dont need to commit this file

**Review by girazoki — APPROVED** [review]
looks good jeremy

**Review by tmpolaczyk — APPROVED** [review]

**Comment by tmpolaczyk** [comment]
So this new weight is not being used, it's only for "documentation" purposes, correct?

**Comment by girazoki** [comment]
> So this new weight is not being used, it's only for "documentation" purposes, correct?

yes for now

**Pull request merged**